In [1]:
import numpy as np
from scipy.constants import k as k_b, m_e

# Dissociative Recombination of $N_2^+$
## Rate Coefficients
The article "Dissociative recombination of N2+, O2+, and NO+: Rate coefficients for ground state and vibrationally excited ions" (https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2003JA010132) gives the rate coefficients to be around $2\times 10^{-7}\,\mathrm{cm^3/s}$ (See table 2).
If we calculate the density of nitrogen in standard conditions and multiply that with the rate coefficients we should get the number of recombinations per sec.

In [2]:
k = 2.0E-7 # cm^3/s from https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2003JA010132
P = 101325.0
T = 293.15

n = P/(k_b*T) * (1/100)**3 # Density in cm^3
print(n)

2.5034768640126714e+19


# Half life
The half life should be $t_{1/2} = \frac{1}{kn}$.

In [3]:
t_half = np.log(2)/(k*n)
print(t_half/1.0E-12)
print(t_half/(1.0E-4*1.0E-12))

0.13843690559395497
1384.3690559395498


The half life of a $N_2^+$ is then 0.13 ps. If the time step in the simulation is 0.1 fs then that is around 1384 time steps.

# Mass
The atomic mass of Nitrogen is 14.0067 u.

In [4]:
m_N = 14.0067*1.6605402E-27 # kg
m_N2 = 2*m_N # Mass of N_2
m_N2p = m_N2 - m_e
print(m_N2)
print(m_N2p)
print(m_N2p/m_e)

4.651737683868e-26
4.6516465900324e-26
51064.339967614666


The mass of $N_2^+$ is then 4.6516465900324e-26 kg or 51064 heavier than the electron.